iocore/net/QUICPacketHandler.cc

```diff
@@ -217,6 +218,23 @@ QUICPacketHandlerIn::_recv_packet(int event, UDPPacket *udp_packet)
   } else {
     vc  = static_cast<QUICNetVConnection *>(qc);
     eth = vc->thread;
+    // qc is not availiable anymore, send stateless reset.
+    // 7.1. Matching Packets to Connections
+    // A server that discards a packet that cannot be associated with a connection MAY also generate a stateless reset 
+    if (vc->in_closed_queue) {
```

Comment:
Please avoid duplicate code. How about changing `if (!qc) {` to `if (!vc || vc->in_closed_queue) {` on line 180.

Reply:
How about movinh the check in front of the `if (!qc) {`,  it seem more clearly.